explore/src/main/scala/explore/observationtree/ObsSummaryTable.scala

```diff
@@ -298,10 +299,12 @@ object ObsSummaryTable extends TableHooks:
         TableStore(props.userId, TableId.ObservationsSummary, cols)
       )
     )
-    .render { (props, ctx, _, _, table) =>
+    .useRef(none[HTMLTableVirtualizer])
```

Comment:
It seems like we don't need the ref to the virtualizer since it's not being used. I think we can safely omit this `useRef` and the `virtualizerRef` parameter below.